Governance/manage/manage-project.md

```diff
@@ -0,0 +1,36 @@
+# Project management
+
+Note, this is NCI specific content - on CSIRO internal resources e.g., access mgmt is governed by windows ACLs - email schelp for advice.
+Project handling   
+On the NCI server disk allocations are managed via projects. Any dataset will be associated to a project, ideally this should be a data specific project, i.e. a project set up to share one or more datasets. It is not ideal to share data via a computational project as this usually are organised by users and have less restrictive permissions.
+If this is unavoidable and the data is hosted on a project with multiple uses, then try to separate the working area from the data collection part.
+
+## proposing a new project
+
+https://opus.nci.org.au/display/Help/How+to+propose+a+new+project
+
+## Mancini or my.nci.org
+
+[Mancini](https://my.nci.org.au/mancini/login?next=/mancini/) is the web interface NCI uses to manage account and allocations. Any NCI users can access it to manage their project memberships. If you are managing a project you gre listed as Lead CI and get access to some extra features.
```

Comment:
"gre"-listed (group+read+execute)

Reply:
may be a typo for 'are'?

Reply:
definitely a typo for "are" this sentence as changed quite a bit, so make sure @Claire you're looking at the latest version and apologies I should have resolved this comment earlier

Reply:
wrong Claire @paolap ;)

Reply:
oops sorry, not sure why GitHub would have suggested your name I should be more careful :-)